[2022-08-29T15:07:59.503-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_python_operator_v04.get_name manual__2022-08-29T19:07:56.518097+00:00 [queued]>
[2022-08-29T15:07:59.520-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_python_operator_v04.get_name manual__2022-08-29T19:07:56.518097+00:00 [queued]>
[2022-08-29T15:07:59.521-0400] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-08-29T15:07:59.521-0400] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2022-08-29T15:07:59.522-0400] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-08-29T15:07:59.566-0400] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): get_name> on 2022-08-29 19:07:56.518097+00:00
[2022-08-29T15:07:59.569-0400] {standard_task_runner.py:52} INFO - Started process 3922 to run task
[2022-08-29T15:07:59.574-0400] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_python_operator_v04', 'get_name', 'manual__2022-08-29T19:07:56.518097+00:00', '--job-id', '2', '--raw', '--subdir', '/home/augenz/PycharmProjects/airflow/dags/dag_with_python_operator_v04.py', '--cfg-path', '/run/user/1000/app/com.jetbrains.PyCharm-Community/tmpvfk8j8z5', '--error-file', '/run/user/1000/app/com.jetbrains.PyCharm-Community/tmpnlk8npig']
[2022-08-29T15:07:59.575-0400] {standard_task_runner.py:80} INFO - Job 2: Subtask get_name
[2022-08-29T15:07:59.754-0400] {task_command.py:371} INFO - Running <TaskInstance: dag_with_python_operator_v04.get_name manual__2022-08-29T19:07:56.518097+00:00 [running]> on host pop-os.localdomain
[2022-08-29T15:07:59.974-0400] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=dag_with_python_operator_v04
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-08-29T19:07:56.518097+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-29T19:07:56.518097+00:00
[2022-08-29T15:07:59.976-0400] {python.py:173} INFO - Done. Returned value was: August
[2022-08-29T15:08:00.078-0400] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=dag_with_python_operator_v04, task_id=get_name, execution_date=20220829T190756, start_date=20220829T190759, end_date=20220829T190800
[2022-08-29T15:08:00.149-0400] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-29T15:08:00.283-0400] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
